### PR TITLE
Add HWMonitor 1.41

### PR DESCRIPTION
**Pull request by xiaoyang-sde**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/784)

### CODE REVIEW

File: manifests/CPUID/HWMonitor/1.41.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+License: Copyright (C) CPUID
+InstallerType: EXE
+AppMoniker: hwmonitor
+Tags: CPU, GPU, HWMonitor, Monitor, Hardware
+Description: HWMonitor is a hardware monitoring program that reads PC systems main health sensors.
+Homepage: https://www.cpuid.com/softwares/hwmonitor.html
+Installers:
+  - Arch: x64
+    Url: http://download.cpuid.com/hwmonitor/hwmonitor_1.41.exe
+    Sha256: 2203e10b9277c0d74c3c12dc6808ea82f581fe01bd78a51b064d413eea9fd149
```
Comment (NathanBnm) [inline manifests/CPUID/HWMonitor/1.41.yaml:14]:
```suggestion
    Sha256: 2203e10b9277c0d74c3c12dc6808ea82f581fe01bd78a51b064d413eea9fd149
    Switches:
      Silent: /VERYSILENT
      SilentWithProgress: /VERYSILENT
```

Comment (NathanBnm) [reply manifests/CPUID/HWMonitor/1.41.yaml:14]: Adding theses switches should fix the issue

**Review by NathanBnm — CHANGES_REQUESTED** [review]

File: manifests/CPUID/HWMonitor/1.41.yaml (hunk 1 above)
Comment (Berkeley-Reject) [reply manifests/CPUID/HWMonitor/1.41.yaml:14]: > Adding theses switches should fix the issue

Thank you very much!

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**